data_portal/viewsets.py

```diff
@@ -36,6 +36,9 @@
     'quality', 'source', 'project_name', 'project_owner', 'experiment_id', 'type', 'assay', 'workflow',
 ]
 
+SEQUENCE_SEARCH_FIELDS = ["id", "instrument_run_id", "run_id", "sample_sheet_name", "gds_folder_path",
```

Comment:
Right, this contain all fields of Sequence model. So we can just use `__all__ ` literal. 

I must admit, I should remove `LIMS_SEARCH_ORDER_FIELDS` and `METADATA_SEARCH_ORDER_FIELDS` as well. Those weren't good. I note myself to refactor them out in next iteration. Please do not follow those pattern.